src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -3,32 +3,44 @@
     using System;
     using System.Collections.Generic;
     using System.Threading;
-    
+
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.ApplicationInsights.DataContracts;
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
     using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation;
-    
+
     /// <summary>
     /// Represents a telemetry processor for sampling telemetry at a fixed-rate before sending to Application Insights.
     /// </summary>
     public sealed class SamplingTelemetryProcessor : ITelemetryProcessor
     {
-        private const string DependencyTelemetryName = "Dependency";
-        private const string EventTelemetryName = "Event";
-        private const string ExceptionTelemetryName = "Exception";
-        private const string PageViewTelemetryName = "PageView";
-        private const string RequestTelemetryName = "Request";
-        private const string TraceTelemetryName = "Trace";
+        private const string DependencyTelemetryName = "DEPENDENCY";
```

Comment:
what happened with CAPSLOCK? :)

Reply:
`ToUpper()` is a requirement for switch cases as per Code Analysis. So I can `ToUpper()` here on lower-case constants or can use a chance to SCREAM in my code :)